hs_core/models.py

```diff
@@ -961,6 +961,36 @@ def __unicode__(self):
         return self.value
 
 
+class FundingAgency(AbstractMetaDataElement):
+    term = 'FundingAgency'
+    agency_name = models.TextField(null=False)
+    award_title = models.TextField(null=True, blank=True)
+    award_number = models.TextField(null=True, blank=True)
+    agency_url = models.URLField(null=True, blank=True)
+
+    class Meta:
+        unique_together = ("content_type", "object_id", "award_number", "agency_name")
```

Comment:
@pkdash @horsburgh OK. Then my only question is whether validation logic needs to be added to check award titles when award_numbers are null. Since this is unlikely to happen, it is not that critical, but there is a possibility that two awards have same funding agency names and same award titles with blank award numbers as implemented currently.